contributors/devel/adding-an-APIGroup.md

```diff
@@ -21,8 +21,11 @@ in your group;
 
 2. Create pkg/apis/`<group>`/{register.go, `<version>`/register.go} to register
 this group's API objects to the encoding/decoding scheme (e.g.,
-[pkg/apis/authentication/register.go](../../pkg/apis/authentication/register.go) and
-[pkg/apis/authentication/v1beta1/register.go](../../pkg/apis/authentication/v1beta1/register.go);
+[pkg/apis/authentication/register.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/authentication/register.go) and
+[pkg/apis/authentication/v1beta1/register.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/authentication/v1beta1/register.go);
+The register files must have a var called SchemeBuilder for the
+generated code to reference. There must be an AddToScheme method for
+the installer to reference.
```

Comment:
agree based on my experiences. Do we have a blurb somewhere about the history of the api group definitions? It is my understanding that `api/` is the old stuff and `apis/` is the new stuff, but I don't have more specific information or understanding.